tests/test_unit_system.py

```diff
@@ -484,6 +484,13 @@ def test_unit_system_init():
         "moment": {"value": 1.0, "units": "N*m"},
         "angular_velocity": {"value": 1.0, "units": "rad/s"},
         "heat_flux": {"value": 1.0, "units": "kg/s**3"},
+        "heat_source": {"value": 1.0, "units": "kg/s**3/m"},
```

Comment:
What is the purpose of defining such units, is it for ensuring that the user is providing the right units when specifying these terms? is the unit system smart enough to do the MLTK dimensional analysis?